sets/hinterlands.py

```diff
@@ -152,6 +154,21 @@ def on_gain(self):
 		self.played_by.deck.append(self)
 		self.game.announce("-- adding " + self.log_string() + " to the top of their deck")
 
+
+class Silk_Road(crd.VictoryCard):
+	def __init__(self, game, played_by):
+		crd.VictoryCard.__init__(self, game, played_by)
+		self.title = "Silk Road"
+		self.description = "Worth 1 Victory Point for every 4 Victory cards in your deck (rounded down)."
+		self.price = 4
```

Comment:
Use card description helpers `"{} for every 4 Victory cards in your deck (rounded down)".format(crd.format_vp(1, true))"'